### ISSUE TITLE
Porting colorbox_swipe

### ISSUE THREAD

**Issue by Graham-72** [backdrop-ops/contrib#224]
I am porting the colorbox_swipe module, full title Colorbox Swipe Gestures Support, from Drupal to Backdrop in order to add Swipe Left and Swipe Right to Colorbox.

**Comment by Graham-72** [comment]
I have a problem because although I have now added this to backdrop-contrib https://github.com/backdrop-contrib/colorbox_swipe and created a release, Github is not creating the Download zip file. Is there something wrong with the info file at https://github.com/backdrop-contrib/colorbox_swipe/blob/1.x-1.0.0/colorbox_swipe.info ?

We used to get a text file if something had gone wrong with this publishing process. Is this still meant to happen?

**Comment by klonos** [comment]
Yes, there was a txt file for some common errors like forgetting to add `type = ` in the .info file. Not sure if this included all errors though.

Silly question, but did you try deleting and re-creating a release?

**Comment by Graham-72** [comment]
@klonos no, that's not a silly question! I must have something wrong somewhere, probably a silly mistake, but deleting tags and hence releases, then creating a new release has not made any difference.

**Comment by jenlampton** [comment]
I don't see anything wrong with the release:
https://github.com/backdrop-contrib/colorbox_swipe/releases/tag/1.x-1.3.0

@klonos there's no attached error messages like there would be if the packager had tried and failed.

@Gormartsen can you check and see if there is something wrong with our packager?

**Comment by klonos** [comment]
@jenlampton yes, that's what I meant by my reply. Sorry this was not clear enough.
